server/chart_config.json

```diff
@@ -1,4 +1,429 @@
 [
+  {
+    "label": "Inequality",
+    "charts": [
+      {
+        "title": "Poverty Rate by Race Across Places",
+        "chartType": "GROUP_BAR",
+        "axis": "PLACE",
+        "statsVars": [
+          "Count_Person_BelowPovertyLevelInThePast12Months_AmericanIndianOrAlaskaNativeAlone",
+          "Count_Person_BelowPovertyLevelInThePast12Months_AsianAlone",
+          "Count_Person_BelowPovertyLevelInThePast12Months_BlackOrAfricanAmericanAlone",
+          "Count_Person_BelowPovertyLevelInThePast12Months_HispanicOrLatino",
+          "Count_Person_BelowPovertyLevelInThePast12Months_NativeHawaiianOrOtherPacificIslanderAlone",
+          "Count_Person_BelowPovertyLevelInThePast12Months_WhiteAlone"
+        ],
+        "denominator": [
+          "Count_Person_AmericanIndianOrAlaskaNativeAlone",
+          "Count_Person_AsianAlone",
+          "Count_Person_BlackOrAfricanAmericanAlone",
+          "Count_Person_HispanicOrLatino",
+          "Count_Person_NativeHawaiianOrOtherPacificIslanderAlone",
+          "Count_Person_WhiteAlone"
+        ],
+        "scaling": 100,
+        "unit": "%",
+        "source": "US Census",
+        "url": "https://www.census.gov/"
+      },
+      {
+        "title": "Poverty Rate by Gender Across Places",
+        "chartType": "GROUP_BAR",
+        "axis": "PLACE",
+        "statsVars": [
+          "Count_Person_Female_BelowPovertyLevelInThePast12Months",
+          "Count_Person_Male_BelowPovertyLevelInThePast12Months"
+        ],
+        "denominator": [
+          "Count_Person_Female",
+          "Count_Person_Male"
+        ],
+        "scaling": 100,
+        "unit": "%",
+        "source": "US Census",
+        "url": "https://www.census.gov/"
+      },
+      {
+        "title": "Income Comparison by Race Across Places",
+        "chartType": "GROUP_BAR",
+        "axis": "PLACE",
+        "statsVars": [
+          "Median_Income_Household_HouseholderRaceAmericanIndianOrAlaskaNativeAlone",
+          "Median_Income_Household_HouseholderRaceAsianAlone",
+          "Median_Income_Household_HouseholderRaceBlackOrAfricanAmericanAlone",
+          "Median_Income_Household_HouseholderRaceHispanicOrLatino",
+          "Median_Income_Household_HouseholderRaceNativeHawaiianOrOtherPacificIslanderAlone",
+          "Median_Income_Household_HouseholderRaceWhiteAlone"
+        ],
+        "source": "US Census",
+        "url": "https://www.census.gov/"
+      },
+      {
+        "title": "Income Comparison by Gender Across Places",
+        "chartType": "GROUP_BAR",
+        "axis": "PLACE",
+        "statsVars": [
+          "Median_Income_Person_15OrMoreYears_Female_WithIncome",
+          "Median_Income_Person_15OrMoreYears_Male_WithIncome"
+        ],
+        "source": "US Census",
+        "url": "https://www.census.gov/"
+      },
+      {
+        "title": "Unemployment Rate by Gender Over Time",
+        "chartType": "LINE",
+        "axis": "TIME",
```

Comment:
Can we show this across places for the overview page?

Reply:
That's what we originally had, but because data for unemployment rate by gender is very limited, you suggested changing the axis to time instead of place

Reply:
ah right :)